content/en/docs/tasks/access-application-cluster/configure-dns-cluster.md

```diff
@@ -2,6 +2,7 @@
 title: Configure DNS for a Cluster
```

Comment:
This will automatically be the H1 and I think it will be the left-nav title too

Reply:
@mistyhacks Thank you for pointing this out!!! Huge Help!